### PR TITLE
Great update!:

### PR DESCRIPTION
**Pull request by Gyx-One**
I.Model accuracy
1.Openpose accuracy finished!: (57.0 Map over random 1160 images over cocoeval 2014 dataset)
  (1)use multiscale search when evaluate (10 Map)
  (2)modified the target generation in training according to openpose caffe source code (2 Map)
  (3)use official pretrained vgg19 backbone (8 Map)
2. Proposal network accuracy update!: (54.6 Pckh over MPII evaluate dataset)
  modified training loss weight according to chainer edition

II.Customise dataset
1.Enable multiple dataset training(eg:MSCOCO and MPII etc,aslong keypoint conversoin is provided)
2.Enable user-add dataset training(eg:user self collected coco-format dataset mixed with official coco dataset)

III.Domain adaptation(testing)
1.Enable domain adaptation in training to anable user augment their model using *Unlabeled* data!

IV.Backbone pretraining
1.Enable pretraining all kinds of model backbones over Imagenet dataset!

### CODE REVIEW

**Review by ganler — APPROVED** [review]
This is awesome!
